content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -32,6 +32,13 @@ You can also use a shorthand alias for `kubectl` that also works with completion
 alias k=kubectl
 complete -o default -F __start_kubectl k
 ```
+Also you can setup the VIM editor. Add below lines in ~/.vimrc
+
+```bash
+set expandtab  ### use spaces for tab
+set tabstop=2  ### amount of spaces used for tab
+set shiftwidth=2 ### amount of spaces used during indentation
+```
```

Comment:
This seems irrelevant to k8s.

Reply:
when we are setting the alias and other exports in .bashrc, then setting the .vimrc is also relevant for some users where they can set the tab as spaces while editing the yaml files. Also spaces for each tab varies from one user to another. So setting it to 2 spaces will make easy for developers to write the yaml files.